VARIABLES.md

```diff
@@ -278,15 +278,15 @@ Default:  false
 
 ### zookeeper_user
 
-Only use to customize Linux User Zookeeper Service runs with. User must exist on host.
+Only use to customize Linux User Zookeeper Service runs with. Default user is cp-kafka
 
 Default:  "{{zookeeper_default_user}}"
 
 ***
 
 ### zookeeper_group
 
-Only use to customize Linux Group Zookeeper Service user belongs to. Group must exist on host.
+Only use to customize Linux Group Zookeeper Service user belongs to. Default group is confluent
```

Comment:
Only use to customize the Linux Group that the Zookeeper Service user belongs to. Default group is confluent.